### PR TITLE
Updating Admin Dashboard

### PR DESCRIPTION
**Pull request by petro97**
Description:

Linting:

- [x] No linting errors

Tests:

- [ ] E2E tests (npm test run with `e2e`)
- [x] Manual tests

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

**Comment by shaunmaharaj** [comment]
@petro97 can you please also ensure you update the version of the components, so they will get pushed to npm

**Comment by petro97** [comment]
> @petro97 can you please also ensure you update the version of the components, so they will get pushed to npm

Yes, I'll update the version of the components

File: app/config/env.js [hunk 1]
```diff
@@ -1,4 +1,24 @@
-'use strict';
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
```
Comment (epmkelly) [inline app/config/env.js]: probably want the copyright dated from 2019 😄

Comment (shaunmaharaj) [reply app/config/env.js]: Good catch @epmkelly 👍

**Comment by shaunmaharaj** [comment]
@petro97 Just a reminder to please increment the version for our component's in package.json for this one 😄

**Comment by petro97** [comment]
> @petro97 Just a reminder to please increment the version for our component's in package.json for this one 😄

Thank you  😄

File: app/src/containers/b2b/AccountMain.less [hunk 2]
```diff
@@ -3,148 +3,96 @@
...
-      padding: 16px 15px;
+  .account-tree-section {
+    width: 250px;
+    flex: 1 1 250px;
+    margin-top: 20px;
+    margin-right: 30px;
+    @media (max-width: 859px) {
+      margin-right: 0;
+    }
+    @media (max-width: @mobileWidth - 1px) {
```
Comment (shaunmaharaj) [inline app/src/containers/b2b/AccountMain.less]: Let's quickly confirm with @BonnieEP if we need to hide this in mobile.

**Comment by shaunmaharaj** [comment]
No further comments. Great work with this @petro97!!!!!

File: app/src/containers/b2b/AccountMain.less (hunk 2 above)
Comment (shaunmaharaj) [reply app/src/containers/b2b/AccountMain.less]: @petro97 Can you please take a look at the AM-UI and follow the same behaviour for displaying the sub-accounts in mobile as that? Will follow up on slack.

**Pull request merged**